docs/app/content-layouts.md

```diff
@@ -0,0 +1,258 @@
+often to stay up-to-date with the latest and greatest. Additionaly, you can join the community on
+[Discord](https://discord.com/invite/directus) and [GitHub](https://github.com/directus/directus/discussions) to find
+thousands of engineers guiding development of the Directus platform.
+
+## How it Works
+
+<video autoplay muted loop controls title="Adjust Item Layouts">
+	<source src="" type="video/mp4" />
+</video>
+
+To adjust an Item's Layout, follow the steps below.
+
+1. Navigate to the Content Module and select the desired Collection.
+2. In the page Sidebar, click **"Layout Options"**.
+3. Choose the desired Layout you want to use.
+4. Customize the Layout as desired.
+
+### Customization Controls
+
+Customization controls can be found in the following three locations.
+
+- **Layout Options** — Located in the Sidebar.
+- **Subheader** — Located just below the Page Header and above the Page Area.
+- **Page Area** — The Page Area, which displays all Items.
+
+These controls typically fall into three general categories.
+
+- **Styling and Formatting**\
+  These are additional customizations to the way a Layout displays such as the size of each Item, how images are cropped,
+  etc.
+
+- **Field Values Displayed**\
+  Most Layouts allow you to choose which Field value(s) are used to represent each Item on the Collection Page. For example,
+  with blog posts, it may be ideal to have the hero image, blog title, date, author, etc.
+
+- **Sorting and Ordering**\
+  Certain Layouts may allow alphabetical and numerical ordering, drag-and-drop adjustments, positioning Items in relation
+  to one another, etc.
+
+### Key Takeaways
+
+- **Layouts will not be universally usable for all Collections.**\
+  Layouts are custom-tailored to support specific data-models. In order to work properly, the Map Layout requires the Collection
+  have a Map Field configured and the Calendar Layout requires the Collection have a datetime Field configured.
+
+- **Controls are different for each Layout.**\
+  Each Layout presents data differently, so certain customizations may not be functional with certain Layouts. For example,
+  the Map Layout displays each Item as a pin on a map, so this Layout has no controls for manual sorting or alphabetical
+  ordering.
+
+- **Controls for a Layout may be in different locations.**\
+  Depending on the Layout, the same control may be under Layout Options in the sidebar, the subheader, or on the Page Area
+  _(and Items themselves)_. For example, the Table Layout lets you set the Field Values displayed in the subheader while
+  the Card Layout lets you set Field Values displayed in the Layout Options menu.
+
+## Table Layout
+
+<video autoplay muted loop controls title="The Table Layout">
+	<source src="" type="video/mp4" />
+</video>
+
+This Layout supports all forms of data, as it directly reflects how data is stored in a Collection. This is the default
+Layout used in the Content Module. It includes the following controls.
```

Comment:
For each Layout:
Each Item will be displayed as a row.
Each Item will be displayed as a Card with an image and optional titles below.
Each Item will be displayed as an event on a calendar.
Each Item will be displayed as a pin on a map.
Each Item will be displayed as a task, assigned under status panels/grouping boards
etc....